packages/psd/src/sections/LayerAndMaskInformation/AdditionalLayerInfo/readPatternAliBlock.ts

```diff
@@ -0,0 +1,189 @@
+// @webtoon/psd
+// Copyright 2021-present NAVER WEBTOON
+// MIT License
+
+import {
+  Point,
+  PatternAliBlock,
+  ColorTable,
+  PatternData,
+  PatternDataChannel,
+  ChannelCompression,
+  ChannelBytes,
+  Pattern,
+} from "../../../interfaces";
+import {Cursor} from "../../../utils";
+import {readPascalString} from "../../ImageResource";
+import {AliBlockBody} from "./AliBlockBody";
+export function readPoint(cursor: Cursor): Point {
+  const vert = cursor.read("u16");
+  const horiz = cursor.read("u16");
+
+  return {vert, horiz};
+}
```

Comment:
I am not sure about this. Point type is used across the project which is defined as 
```export type Point = {vert: number; horiz: number};```

Reply:
added a line above the function though :)

Reply:
We could just return `[width, height]` since we destructure it right after call anyway: https://github.com/opendesigndev/psd-ts/blob/ee4c1b5128a988cc21841c87f5255d067ba7fecc/packages/psd/src/sections/LayerAndMaskInformation/AdditionalLayerInfo/readPatternAliBlock.ts#L131

Reply:
changed